udfs/community/test_cases.js

```diff
@@ -452,6 +452,43 @@ generate_udf_test("url_param", [
         expected_output: `NULL`
     },
 ]);
+generate_udf_test("url_trim_query", [
```

Comment:
Add test cases for NULL inputs and a test case for url which contains keys but no hash